src/App.js

```diff
@@ -1,5 +1,108 @@
+const MissionUtils = require("@woowacourse/mission-utils");
+
 class App {
-  play() {}
+  constructor() {
+    this.computerNumberArray = this.getComputerRandomNumber();
+    this.userInputNumber;
+    this.countStrike = 0;
+    this.countBall = 0;
+    MissionUtils.Console.print('숫자 야구 게임을 시작합니다.');
+  }
+
+  getComputerRandomNumber() {
+    const computerNumberSet = new Set();
+    while (computerNumberSet.size < 3) {
+      computerNumberSet.add(MissionUtils.Random.pickNumberInRange(1,9));
+    }
+    const computerNumberArray = [...computerNumberSet];
+    return computerNumberArray;
+  }
+
+  play() {
+    MissionUtils.Console.readLine('숫자를 입력해주세요 :', (userInputNumber) => {
+      this.userInputNumber = userInputNumber;
+      this.isError();
+      this.count();
+      this.printMessage();
+      if (this.countStrike < 3) {
+        this.countStrike = 0;
+        this.countBall = 0;
+        this.play(); 
+      }
+      else if (this.countStrike === 3) {
+        MissionUtils.Console.print('3개의 숫자를 모두 맞히셨습니다! 게임 종료');
+        this.isGameRestart();
+      }
+    });
+  }
+
+  count() {
+    const userNumberArray = this.userInputNumber.split('');  
+    const computerStringArray = this.computerNumberArray.map(num=>num.toString());
+    
+    for ( let i=0; i<3; i++ ) {
+      if ( userNumberArray[i] === computerStringArray[i] ) {
+        this.countStrike += 1;
+      }
+      else if (userNumberArray.includes(computerStringArray[i])) {
+        this.countBall += 1;
+      }
+    }
+  }
```

Comment:
```suggestion
  count() {
    const userNumberArray = this.userInputNumber.split('');  
    const computerStringArray = this.computerNumberArray.map(num=>num.toString());
    let countStrike = 0;
    let countBall = 0;
    
    for ( let i=0; i<3; i++ ) {
      if ( userNumberArray[i] === computerStringArray[i] ) {
        countStrike += 1;
      }
      else if (userNumberArray.includes(computerStringArray[i])) {
        countBall += 1;
      }
    }
    return {countStrike, countBall};
  }
```

반환값을 사용해주면 매번 초기화하지 않아도 될것같습니다 !